pilot/lib/pilot/build_manager.rb

```diff
@@ -336,6 +338,10 @@ def should_update_localized_build_information?(options)
       !options[:localized_build_info].nil?
     end
 
+    def should_update_build_beta_details?(options)
+      options[:notify_external_testers]
+    end
```

Comment:
Is this method going to get reused in the future? If not... I think I’d prefer this logic to just be moved into the if statement on line 256 since this is pretty specific to that call of `update_build_beta_details` 😊

Reply:
@joshdholtz sure! I was just following what I thought to be the convention, writing these should_* methods. I'll update the patch asap, thx!